Cabal/src/Distribution/Simple/GHC.hs

```diff
@@ -139,12 +139,12 @@ configure verbosity hcPath hcPkgPath conf0 = do
       (userMaybeSpecifyPath "ghc" hcPath conf0)
   let implInfo = ghcVersionImplInfo ghcVersion
 
-  -- Cabal currently supports ghc >= 7.0.1 && < 9.4
+  -- Cabal currently supports ghc >= 7.0.1 && < 9.6
   -- ... and the following odd development version
-  unless (ghcVersion < mkVersion [9,4]) $
+  unless (ghcVersion < mkVersion [9,6]) $
     warn verbosity $
          "Unknown/unsupported 'ghc' version detected "
-      ++ "(Cabal " ++ prettyShow cabalVersion ++ " supports 'ghc' version < 9.4): "
+      ++ "(Cabal " ++ prettyShow cabalVersion ++ " supports 'ghc' version < 9.6): "
```

Comment:
Can this constant (now `9.6`) be factored out so that there is a single place where it has to be changed?